Source/Particles/Collision/CollisionType.H

```diff
@@ -12,6 +12,13 @@
 #include <AMReX_REAL.H>
 #include <AMReX_ParmParse.H>
 
+namespace findParticles {
+    amrex::DenseBins<WarpXParticleContainer::ParticleType>
+    findParticlesInEachCell( int const lev, amrex::MFIter const& mfi,
+                             WarpXParticleContainer::ParticleTileType const& ptile);
+}
```

Comment:
I needed to use the function `findParticlesInEachCell ()`. However, it was defined in an unnamed namespace in `CollisionType.cpp` so I couldn't directly reuse it in another file. My quick solution was to give a name to the namespace and declare it in the header file `CollisionType.H`.

Do we want to leave this as it is? It seems to me that this function is more general than just collisions so it might make sense to move it to a more general place.

Reply:
I agree with you, putting it in a named namespace is better, maybe a namespace `ParticleUtils` in `Source/Utils`? Maybe other particle functions would naturally go there too.

Reply:
Sounds good :+1: . Do you prefer doing the change here or in a separate PR?

Reply:
If you identify other functions that belong to `ParticleUtils`, another PR would be better. Otherwise, just include the change in this PR.

Reply:
I did not see any other function that obviously belongs to `ParticleUtils` so I've only put this one in a new namespace in `Utils/ParticleUtils.cpp/H`.

(In fact most particle functions already have their own file in an appropriate location. One could argue that structs like `GetParticlePosition` and `SetParticlePosition` defined in `Particles/Pusher/GetAndSetPosition.H` are more general than just pusher and could also belong in `ParticleUtils`, but they currently don't really feel misplaced either)